src/containers/Footer/footer.scss

```diff
@@ -1,69 +1,64 @@
 @import '../shared/css/variables.scss';
 
 .footer {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  border-top: 1px solid $black-80;
+  margin: 0 10%;
   background: $black;
-  .footer-content {
```

Comment:
What exactly was `.footer-content` doing, and what did we do with the attributes that were  specified here (padding/borders) that were wrapping the `.footer-links`

Reply:
i completely re-styled the whole footer, so that answers you on the attributes. the `footer-content` had no purpose as an html element, just unnecessary depth to the tree